src/brain_atlas/util/hail_batch/14_hail_docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mambaorg/micromamba:0.24.0
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER brain-atlas /brain-atlas
+
+RUN cd /brain-atlas \
+       && micromamba install -n base -y -f environment.yml \
+       && micromamba clean --all --yes
+
+# need to install protobuf manually bc broken for some reason
+# https://github.com/spesmilo/electrum/issues/7825
+ARG MAMBA_DOCKERFILE_ACTIVATE=1 
+RUN pip install --no-cache-dir protobuf==3.20.1; 
+
+COPY  --chown=$MAMBA_USER:$MAMBA_USER  getcpu.sh       /getcpu.sh
+# COPY --chown=$MAMBA_USER:$MAMBA_USER killallpython3 /killallpython3
+
+USER root
+# need bc for getcpu.sh
+# procps so can remove all processes to force shutdown (ps and top)
+RUN apt-get -y update \
+    && apt-get -y install bc procps tmux sudo \
+    && apt-get -y clean
+    
+
+# hail batch fails internally if not run as root, seems to work fine with conda
+# USER $MAMBA_USER
+USER root
+
+RUN adduser $MAMBA_USER sudo \
+     && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
the mamba docs very strongly want to use `$MAMBA_USER`, especially for the conda package installation.

But then there's weirdness where hail batch assumes you run as root, and their internal setup code fails if you don't (I could post an issue with them :shrug: ). So not elegant. You can use anaconda and side step all of this it's just a good amount slower to create the initial docker image (not load in I mean) and I preferred mamba.

The sudo line might not be necessary with the current config, just the `USER root` line. As a tangent, I was having trouble with hail batch keeping jobs open even after the python finished, because the dask forks/children weren't closing. So I needed mamba to have sudo to do increasingly aggressive `kill` lines. But then `sys.exit` worked when I did that

Reply:
Oh ok or maybe I just hallucinated. 
https://github.com/mamba-org/micromamba-docker
> Running with --user=root is supported.

I somehow read that as "is _not_ supported". So maybe it call all be run as root, either way

Reply:
since there's always a `USER root` command on line 17 and the MAMBA_USER isn't used, I think these lines can just be removed.

Reply:
sure ok, do you want me to push a commit removing them?

Just for ease of context switching ease, I can collect the more cosmetic changes and make a bigger commit later, if that works, as long as the bones of the parallelization end up working on your end

Reply:
I can do it, was just double-checking that it wasn't doing something important.